### PR TITLE
Update vit flax test

### PR DESCRIPTION
**Pull request by aireenmei**

### CODE REVIEW

**Comment by aireenmei** [comment]
@will-cromar

**Comment by aireenmei** [comment]
TESTED=flax-latest-vit-func-v2-8-1vm-f2h4v(https://pantheon.corp.google.com/kubernetes/job/us-central1/oneshots-us-central1/default/flax-latest-vit-func-v2-8-1vm-f2h4v/details?organizationId=433637338589&project=xl-ml-test)

File: tests/jax/latest/vit.libsonnet [hunk 1]
```diff
@@ -41,23 +41,30 @@ local tpus = import 'templates/tpus.libsonnet';
...
+    git clone https://github.com/google-research/vision_transformer.git
+    cd vision_transformer
+    pip install -r vit_jax/requirements.txt
+    cd ..
+    git clone https://github.com/google/flaxformer.git
+    cd flaxformer
+    pip3 install '.[testing]'
+    cd ../vision_transformer    
+
+    pip install --upgrade jaxlib jax flax clu %(extraDeps)s
```
Comment (sshahrokhi) [inline tests/jax/latest/vit.libsonnet:54]: why are you installing jaxlib and jax twice? (that installLatest jax and jaxlib should already install jax and jaxlib?)

Comment (sshahrokhi) [reply tests/jax/latest/vit.libsonnet:54]: I guess if in the requirements wrong versions are installed, maybe we can let the owners know that the requirements are wrong.

**Comment by sshahrokhi** [comment]
never mind, you made another PR.

**Pull request closed without merging**